### PR TITLE
Add some padding and fix a border bug.

### PR DESCRIPTION
**Pull request by adamjspooner**
@keybase/react-hackers, this is ready for review.

### CODE REVIEW

File: shared/chat/conversation/messages/wrapper/index.js [hunk 1]
```diff
@@ -541,11 +541,13 @@ const styles = Styles.styleSheetCreate({
...
     isElectron: {
-      border: `1px solid ${Styles.globalColors.black_10}`,
+      borderBottom: `1px solid ${Styles.globalColors.black_10}`,
       borderBottomLeftRadius: Styles.borderRadius,
       borderBottomRightRadius: Styles.borderRadius,
-      borderTopWidth: 0,
+      borderLeft: `1px solid ${Styles.globalColors.black_10}`,
+      borderRight: `1px solid ${Styles.globalColors.black_10}`,
       bottom: -24,
+      paddingBottom: 8,
```
Comment (songgao) [inline shared/chat/conversation/messages/wrapper/index.js]: Could use `Styles.globalMargins.tiny`? Same below

File: shared/chat/conversation/messages/wrapper/index.js [hunk 2]
```diff
@@ -541,14 +541,16 @@ const styles = Styles.styleSheetCreate({
   ellipsis: {marginLeft: Styles.globalMargins.tiny},
   emojiRow: Styles.platformStyles({
     isElectron: {
-      border: `1px solid ${Styles.globalColors.black_10}`,
+      borderBottom: `1px solid ${Styles.globalColors.black_10}`,
       borderBottomLeftRadius: Styles.borderRadius,
       borderBottomRightRadius: Styles.borderRadius,
-      borderTopWidth: 0,
```
[the remark this answers is not in the document]
Comment (songgao) [reply shared/chat/conversation/messages/wrapper/index.js]: I'm a bit curious why this only happens "sometimes"

[the remark this answers is not in the document]
Comment (adamjspooner) [reply shared/chat/conversation/messages/wrapper/index.js]: Yeah, it was weird. The inspector definitely showed `border-top-width: 1px`.

File: shared/chat/conversation/messages/wrapper/index.js (hunk 1 above)
Comment (adamjspooner) [reply shared/chat/conversation/messages/wrapper/index.js]: Thanks!

**Pull request merged**